apps/mochi-web/components/explore/index/sections/SummarySection.tsx

```diff
@@ -0,0 +1,170 @@
+import { utils } from '@consolelabs/mochi-ui'
+import { Skeleton } from '@mochi-ui/core'
+import {
+  DollarColored,
+  HeartColored,
+  LinkColored,
+  UserShieldColored,
+} from '@mochi-ui/icons'
+import { useEffect } from 'react'
+import { Pie, PieChart } from 'recharts'
+import { formatNumber } from '~utils/number'
+import { useTransactionSummaryStore } from '../stores'
+
+export const SummarySection = () => {
+  const {
+    loading: isTransactionSummaryLoading,
+    transactionSummary,
+    fetchTransactionSummary,
+  } = useTransactionSummaryStore()
+
+  useEffect(() => {
+    fetchTransactionSummary()
+  }, []) // eslint-disable-line
+
+  const loading = isTransactionSummaryLoading || !transactionSummary
+
+  return (
+    <div className="bg-neutral-100 p-3 lg:py-14 lg:px-30">
```

Comment:
Try to use color codes from `semantics.ts` for later darkmode support
```suggestion
    <div className="bg-background-level2 p-3 lg:py-14 lg:px-30">
```